### PR TITLE
Adding testcase for AssociateLabelDisplay-MNRVA-504

### PR DESCRIPTION
**Pull request by RiaRenold**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-504

 ## Description:
As part of MNRVA-504 testing added one testcase to AssociateLabel-MNRVA-504 branch.Please review.
 
## Testing:
npm run e2e
 
## Screenshots:
(if applicable)

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -6,10 +6,12 @@ const { SpecReporter } = require("jasmine-spec-reporter");
 exports.config = {
   allScriptsTimeout: 11000,
   specs: ["./src/**/*e2e-spec.ts"],
-  capabilities: {
+  // specs:[".src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts"],
+  // specs:["e2e/src/features/monitoring/monitorlist/monitorsPagination.e2e-spec.ts"],
+    capabilities: {
     browserName: "chrome",
     chromeOptions: {
-      args: ["--no-sandbox", "--headless", "--disable-gpu", "--disable-web-security"],
```
Comment (pras7430) [inline e2e/protractor.conf.js:12]: we should have "headless"  in containers (stage env), 
So we can remove it while writing the test cases and add it back while pushing the code.

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { AppPage } from "../../../pages/app.po";
+import { navigations } from "../../../commons/navigations";
+import { monitorsListPage } from "../../../pages/monitorlistpage";
+import { monitorsDetailsPage } from "../../../pages/monitordetailspage";
+import { browser } from "protractor";
+// import { default as obj } from "../../../../object.json";
```
Comment (pras7430) [inline e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:6]: hey, hope we are not going to use this line in future

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      page1.monitorName().click();
+      browser.sleep(5000);
+      page2 = new monitorsDetailsPage();
+
+   });
+
+   var labelKeys = Object.keys(obj.labelSelector);
+   console.log(labelKeys);
+
+   using([{ key: labelKeys[0], value: obj.labelSelector[labelKeys[0]]}, { key: labelKeys[1], value: obj.labelSelector[labelKeys[1]] },
```
Comment (pras7430) [inline e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:40]: Nice !!

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { browser } from "protractor";
+// import { default as obj } from "../../../../object.json";
+import { default as using } from "../../../../../node_modules/jasmine-data-provider";
+import { default as obj } from "../../../../../src/app/_mocks/monitors/single.json";
+
+
+
+
+
+describe("Test label field display on Monitor's details page", function () {
```
Comment (pras7430) [inline e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:14]: we can use fat arrow functions

File: e2e/object.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (pras7430) [inline e2e/object.json:1]: Hey, you can remove this file, As it is not in use now.

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts (hunk 3 above)
Comment (pras7430) [reply e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:40]: "console log" is here

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts (hunk 4 above)
Comment (FarajiA) [reply e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:14]: I love fat arrows!!

File: e2e/object.json (hunk 5 above)
Comment (RiaRenold) [reply e2e/object.json:1]: Okay.

File: e2e/protractor.conf.js (hunk 1 above)
Comment (RiaRenold) [reply e2e/protractor.conf.js:12]: okay,i get it

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts (hunk 4 above)
Comment (RiaRenold) [reply e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:14]: noted:)

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts (hunk 2 above)
Comment (RiaRenold) [reply e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:6]: oh yes,

File: e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts (hunk 3 above)
Comment (RiaRenold) [reply e2e/src/features/monitoring/monitordetails/monitordetailslabelinfodisplay.e2e-spec.ts:40]: will be removed in the next push.

**Pull request merged**